src/main/java/com/xpanxion/java/assignments/student5/Worker5.java

```diff
@@ -1,8 +1,25 @@
 package com.xpanxion.java.assignments.student5;
 
+import com.xpanxion.java.assignments.DataAccess;
+import com.xpanxion.java.assignments.model.Product;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class Worker5 {
 
     public void ex1() {
-        System.out.println("Ex. 1...");
+        List<Product> products = DataAccess.getProducts();
+        Map<Integer, String> departmentNames = new HashMap<>();
+
+        DataAccess.getDepartments().stream()
```

Comment:
I know this works but it still looks very odd to me.  Can we try to follow this approach?

https://howtodoinjava.com/java8/collect-stream-to-map/